sema/commons/j2/syntax_builder.py

```
@@ -66,13 +66,21 @@ def variables_in_template(self, name: str) -> set:  # type: ignore
         ast = self._templates_env.parse(*template_source)  # type: ignore
         return meta.find_undeclared_variables(ast)
 
-    def build_syntax(self, name: str, **variables) -> str:  # type: ignore
+    def build_syntax(
+        self,
+        _template_name: str,
+        /,
+        **variables,
```

Ruff (0.8.2)

Missing type annotation for `**variables`

(ANN003)